### DISCUSSION TITLE
Optimistic Voting

### DISCUSSION THREAD

**Discussion by JakeHartnell** [DA0-DA0/dao-contracts#468] [General]
Many DAOs in the Ethereum ecosystem are starting to explore "Optimistic Voting" to combat voting fatigue and reducing gas fees spent for on-chain votes. The principal is simple, assume a proposal will pass _unless_ people vote no.

We could easily support this with a custom proposal module in DAO DAO, does this interest people at all?

**Comment by 0xekez** [comment]
I'm personally pretty skeptical of optimistic voting. If people are worried about voter fatigue, maybe we can direct them to:

1. Create subDAOs to do things that they don't want to require the whole DAO to vote on.
2. Create those subDAOs with `min_voting_period` set to longer than the parent DAO's voting period.

Having done this, the parent DAO will always be able to act more quickly than the subDAO to revoke privileges if they'd like to "say no".

**Comment by baoskee** [comment]
I think some DAOs should be biased towards preservation rather than change. Maybe have a setting for pessimistic vs. optimistic voting. Also I have a hypothesis that most votes will be delegated in the future, with exceptions.
